docs/wallets.md

```diff
@@ -0,0 +1,41 @@
+---
+id: wallets
+title: Polkadot Wallets
+sidebar_label: Polkadot Wallets
+---
+
+## Polkadot Wallets
+
+The integration of a [wallet](https://wiki.polkadot.network/docs/en/glossary#wallet) and Polkadot
+allows for simple and easy access to private keys and signing transactions. Below are wallets
+supported by Polkadot.
+
+### Live Wallets
+
+- [Polkawallet](https://polkawallet.io/) - Mobile wallet available on IOS and Android
+- [Math Wallet](https://www.mathwallet.org/en-us/) - Browser extension and mobile wallet
+- [Cobo Wallet](https://cobo.com/) - Mobile wallet
+
+### Currently in Development
+
+- [Lunie](https://lunie.io/) - Browser extension wallet. Follow developement on
+  [GitHub](https://github.com/luniehq/Web3-collaboration)
+- [Zondax](https://zondax.ch/) - Ledger app for Polkadot. Development on
+  [GitHub](https://github.com/ZondaX)
+- [ImToken](https://token.im/) - Mobile wallet available on IOS and Android
+- [BlockX Labs](https://blockxlabs.com/) - Browser extension wallet, called Enzyme. Follow
+  development on [GitHub](https://github.com/blockxlabs/enzyme/)
+- [Speckle OS](https://www.speckleos.io/) - Browser extension wallet. Progress found on
+  [GitHub](https://github.com/GetSpeckle/speckle-browser-extension)
+- [AirGap](https://airgap.it/) - Desktop and mobile wallet. Development found on
+  [GitHub](https://github.com/airgap-it/airgap-wallet)
+- [NGRAVE](https://www.ngrave.io/) - Hardware wallet
+- [Trust Wallet](https://trustwallet.com/) - Mobile app available on IOS and Android
+- [AToken](https://www.atoken.com/) - Mobile app available on IOS and Android
+- [Qbao Network](https://qbao.fund/) - Mobile app available on IOS and Android
+- [Node Factory](https://nodefactory.io/) - Metamask wallet plugin
+
+### Wallets still being Explored
+
+- [Token Pocket](https://www.tokenpocket.pro/en/) - Mobile and desktop wallet
+- [Atomic Wallet](https://atomicwallet.io/) - Mobile and desktop wallet 
```

Comment:
If any of the wallets say that we haven't gotten in touch with them recently, let's not add them quite yet.